lesson-7/public/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+ Name: <input type="TEXT" id="nameInput" />
+ Comment:  <input type="TEXT" id="commentInput" />
+  <input type="SUBMIT" id="button" />
+  <div>
+    <pre id="chat" />
+    </div>
+<script>
+const button = document.getElementById('button');
+button.onclick =() =>{
+  const name = document.getElementById('nameInput').value;
+  const comment = document.getElementById('commentInput').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('post', '/submit');
+  xhttp.setRequestHeader('content-type','application/json');
+  xhttp.send(JSON.stringify({
+    name,
```

Comment:
I didn't know it could be done this way. Nice!

Reply:
if you are doing this way, it might make sense to put in 1 line: `stringify({name, comment})`